src/filename_inspector.py

```diff
@@ -0,0 +1,106 @@
+import fnmatch
+import glob
+import os
+import json
+import csv
+
+
+def get_input(name: str) -> str:
+    value = os.getenv(f'INPUT_{name.upper()}')
+    return value
+
+
+def set_output(name: str, value: str):
+    output_file = os.getenv('GITHUB_OUTPUT', None)
+    if output_file:
+        with open(output_file, 'a') as f:
+            f.write(f'{name}={value}\n')
+    else:
+        raise EnvironmentError('GITHUB_OUTPUT environment variable is not set.')
```

Comment:
this looks like a precondition we should check for at the beginning of the script

Reply:
Reworked and tested in commits:
- 16039bf97629247624e790995c7fe698d2cf7094
- 857e388dd30621c4f4bb84aa4c3371453ebf3f61